generators/__init__.py

```diff
@@ -17,7 +17,7 @@
 )
 
 from .reference_quality import (
-    tiktoken_token_counter
+    
 )
```

Comment:
We need to remove this import in total, otherwise we run into a syntax error

Reply:
Oh yes, removed it!